racetrack_client/racetrack_client/utils/semver.py

```diff
@@ -70,6 +70,11 @@ def find_latest_wildcard(
         latest_version = max(versions_objects.keys())
         return versions_objects[latest_version]
 
+    @property
+    def is_stable(self) -> bool:
+        """Return true, if it's not a pre-release version - has no pre-release label."""
+        return not bool(self.label)
```

Comment:
This also works

```suggestion
        return not self.label
```

![image](https://github.com/TheRacetrack/racetrack/assets/124889668/10ec4d5f-01c3-4991-82ec-651e13ce7129)